components/nav/NavFooter.vue

```diff
@@ -38,3 +43,20 @@ function toggleDark() {
     </div>
   </footer>
 </template>
+
+<style>
+  .beta-label {
+    width: 212px;
+    height: 70px;
+    background-color: var(--c-primary);
+    border: 2px solid var(--c-primary);
+    border-radius: 8px;
+    padding: 8px;
+    gap: 8px;
+    line-height: 17.8px;
+
+    color: var(--c-text-base-light);
+    font-size: 13.13px;
+    font-weight: 500;
+  }
+</style>
```

Comment:
Similar question re: UnoCss.  Just want to be wary of mixing patterns, even if another pattern is more comfortable to work in

Reply:
Aly helped to remove lots of these rules, and I removed a couple more